conjure-java-core/src/integrationInput/java/com/palantir/product/ErrorServiceBlocking.java

```diff
@@ -0,0 +1,386 @@
+            public TestEmptyBodyResponse testEmptyBody(AuthHeader authHeader) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams("Authorization", authHeader.toString());
+                return _runtime.clients()
+                        .callBlocking(testEmptyBodyChannel, _request.build(), testEmptyBodyDeserializer);
+            }
+
+            @Override
+            public TestBinaryResponse testBinary(AuthHeader authHeader) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams("Authorization", authHeader.toString());
+                return _runtime.clients().callBlocking(testBinaryChannel, _request.build(), testBinaryDeserializer);
+            }
+
+            @Override
+            public TestOptionalBinaryResponse testOptionalBinary(AuthHeader authHeader) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams("Authorization", authHeader.toString());
+                return _runtime.clients()
+                        .callBlocking(testOptionalBinaryChannel, _request.build(), testOptionalBinaryDeserializer);
+            }
+
+            @Override
+            public String toString() {
+                return "ErrorServiceBlocking{_endpointChannelFactory=" + _endpointChannelFactory + ", runtime="
+                        + _runtime + '}';
+            }
+        };
+    }
+
+    /** Creates an asynchronous/non-blocking client for a ErrorService service. */
+    static ErrorServiceBlocking of(Channel _channel, ConjureRuntime _runtime) {
+        if (_channel instanceof EndpointChannelFactory) {
+            return of((EndpointChannelFactory) _channel, _runtime);
+        }
+        return of(
+                new EndpointChannelFactory() {
+                    @Override
+                    public EndpointChannel endpoint(Endpoint endpoint) {
+                        return _runtime.clients().bind(_channel, endpoint);
+                    }
+                },
+                _runtime);
+    }
+
+    final class Factory implements DialogueServiceFactory<ErrorServiceBlocking> {
+        @Override
+        public ErrorServiceBlocking create(EndpointChannelFactory endpointChannelFactory, ConjureRuntime runtime) {
+            return ErrorServiceBlocking.of(endpointChannelFactory, runtime);
+        }
+    }
+
+    sealed interface TestBasicErrorResponse
+            permits TestBasicErrorResponse.Success, TestBasicErrorResponse.InvalidArgument {
+        record Success(String value) implements TestBasicErrorResponse {
+            public Success {
+                Preconditions.checkArgumentNotNull(value, "value cannot be null");
+            }
+        }
+
+        final class InvalidArgument extends ConjureErrors.BaseEndpointError<TestErrors.InvalidArgumentParameters>
+                implements TestBasicErrorResponse {
+            @JsonCreator
+            InvalidArgument(
```

Comment:
let's generate log-safety annotations for these based on the safety of the *Parameters type